### ISSUE TITLE
Invalid Authorization header causing errors

### ISSUE THREAD

**Issue by doitadrian** [webiny/webiny-js#1300]
## This is:

- Bug

## Expected Behavior
I should just get auth error when sending this:
![image](https://user-images.githubusercontent.com/5121148/95648415-26322900-0ad7-11eb-85a8-36c2ddcc1882.png)

## Actual Behavior
```2020-10-10T06:49:57.856Z	b455b200-9a98-43ac-ab8a-1560d6758683	INFO	dobeo res {
  statusCode: 500,
  body: `{"error":{"name":"TypeError","message":"Cannot destructure property 'header' of '_jsonwebtoken.default.decode(...)' as it is null.","stack":"TypeError: Cannot destructure property 'header' of '_jsonwebtoken.default.decode(...)' as it is null.\\n    at Object.authenticate (/var/task/handler.js:59191:11)\\n    at async Object.apply (/var/task/handler.js:59065:24)\\n    at async Runtime.handler (/var/task/handler.js:59647:9)"}}`,
  headers: { 'Cache-Control': 'no-store', 'Content-Type': 'application/json' }
}
```

**Comment by daniel-shuy** [comment]
I've created a PR to fix this in `v5` at #1330, and backported the fix for `v4` at #1331

**Comment by daniel-shuy** [comment]
hi @doitadrian, could you review the PRs?
